### PR TITLE
Raise exception with stderr

### PR DESCRIPTION
**Pull request by dmahmouali**
issue : https://github.com/threefoldtech/js-sdk/issues/103

### CODE REVIEW

File: jumpscale/tools/git/__init__.py [hunk 1]
```diff
@@ -78,11 +78,15 @@ def clone_repo(url: str, dest: str, branch_or_tag="", depth=0, commit_id=""):
         cmd += f" -b {branch_or_tag}"
     if depth != 0:
         cmd += f" --depth={depth}"
-    j.core.executors.run_local(cmd)
+    rc, _, err = j.core.executors.run_local(cmd, warn=True)
+    if rc > 0:
+        raise j.exceptions.Runtime(err)
```
Comment (xmonader) [inline jumpscale/tools/git/__init__.py:83]: please mention the command you were trying to run in the error message

File: jumpscale/tools/git/__init__.py [hunk 2]
```diff
@@ -78,11 +78,15 @@ def clone_repo(url: str, dest: str, branch_or_tag="", depth=0, commit_id=""):
...
     repo_name = j.sals.fs.basename(url).split(".git")[0]
     if commit_id:
         prefix = f"cd {dest}/{repo_name} && "
-        j.core.executors.run_local(prefix + "git checkout {commit_id}")
+        rc, _, err = j.core.executors.run_local(prefix + f"git checkout {commit_id}", warn=True)
+        if rc > 0:
+            raise j.exceptions.Runtime(err)
```
Comment (xmonader) [inline jumpscale/tools/git/__init__.py:89]: please make to record the command you were trying to execute along with the stderr

File: jumpscale/tools/git/__init__.py (hunk 1 above)
Comment (dmahmouali) [reply jumpscale/tools/git/__init__.py]: edited stderr :+1:, thanks

File: jumpscale/tools/git/__init__.py (hunk 2 above)
Comment (dmahmouali) [reply jumpscale/tools/git/__init__.py]: edited stderr :+1: thanks

**Pull request merged**